src/deploy/enum_event_category.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy maevsi:enum_event_category to pg
+
+BEGIN;
+
+CREATE TYPE maevsi.event_category AS ENUM (
```

Comment:
Let's create a table for that, so some administrators can change the list via some content management system in the future without having to change the database code.